### PR TITLE
Adding support for avaya_ers_show_logging_config command

### PR DESCRIPTION
**Pull request by kadecole**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

avaya_ers_show_logging_config

##### SUMMARY

Adding new template for Avaya ERS for show logging config command

```

```

### CODE REVIEW

File: templates/avaya_ers_show_logging_config.template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Value REMOTE_ADDRESS (\S+)
+Value SECONDARY_ADDRESS (\S+)
+Value REMOTE_TYPES ([\S ]*)
+Value FACILITY (\S+)
+
+Start
+  ^Event Logging:\s+${EVENT_LOGGING}
+  ^Volatile Logging Option:\s+${VOLATILE}
+  ^Event Types To Log:\s+${LOG_TYPES}
+  ^Event Types To Log To NV Storage:\s+${NV_STORAGE_TYPES}
```
Comment (jmcgill298) [inline templates/avaya_ers_show_logging_config.template]:
```suggestion
  ^Event Types To Log To NV Storage:\s+${NV_STORAGE_TYPES}\s*$$
```

**Comment by kadecole** [comment]
Thanks for the code review. I have updated the PR with some of the changes requested.

I based this template off previous templates and they showed Done on the last line of the file. If this Done is not needed I will remove.

Also for the other code review suggestions, I am matching on what should be returned. There should only be two options for the text returning in those fields.

File: templates/avaya_ers_show_logging_config.template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Value EVENT_LOGGING (Enabled|Disabled)
```
[the remark this answers is not in the document]
Comment (kadecole) [reply templates/avaya_ers_show_logging_config.template]: I am matching on what should be returned. There should only be two options for the text returning in those fields.

File: templates/avaya_ers_show_logging_config.template [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Value VOLATILE (Latch|Overwrite)
```
[the remark this answers is not in the document]
Comment (kadecole) [reply templates/avaya_ers_show_logging_config.template]: I am matching on what should be returned. There should only be two options for the text returning in those fields.

File: templates/avaya_ers_show_logging_config.template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Value LOG_TYPES ([\S ]*)
+Value NV_STORAGE_TYPES ([\S ]*)
+Value REMOTE_LOGGING (Enabled|Disabled)
```
[the remark this answers is not in the document]
Comment (kadecole) [reply templates/avaya_ers_show_logging_config.template]: I am matching on what should be returned. There should only be two options for the text returning in those fields.

**Comment by kadecole** [comment]
I have updated from your suggestions. Please review and merge when possible. Thanks.

**Pull request merged**